header.tpl

```diff
@@ -188,7 +188,7 @@
           </div>
           <div class="menu-item">
             <img class="top-icon" src="img/update-icon.svg" alt="update icon"/>
-            <a href="https://github.com/NullHypothesis/censorbib/commits/master">Last update: 2024-02-17</a>
+            <a href="https://github.com/NullHypothesis/censorbib/commits/master">Last update: 2024-02-25</a>
```

Comment:
You leaked a few bits of your geolocation because it's still 2024-02-24 where I live 😉